### ISSUE TITLE
GET /

### ISSUE THREAD

**Issue by DronMDF** [DronMDF/zold-lazy-node#3]
```
{
  "version": "0.6.1" ,
  "score": {
    "value": 3,
    "time":  "2017-07-19T21:24:51Z",
    "host": "b2.zold.io",
    "port": 4096,
    "invoice": "THdonv1E@0000000000000000",
    "suffixes": [ "4f9c38" , "49c074" , "24829a" ],
    "strength": 6
  }
}
```
В заголовке http должна быть следующая информация:
Content-Type , Content-Length , and X-Zold-Version

**Comment by 0pdd** [comment]
@DronMDF 3 puzzles [#29](https://github.com/DronMDF/zold-lazy-node/issues/29), [#30](https://github.com/DronMDF/zold-lazy-node/issues/30), [#31](https://github.com/DronMDF/zold-lazy-node/issues/31) are still not solved.

**Comment by 0pdd** [comment]
@DronMDF 3 puzzles [#30](https://github.com/DronMDF/zold-lazy-node/issues/30), [#31](https://github.com/DronMDF/zold-lazy-node/issues/31), [#41](https://github.com/DronMDF/zold-lazy-node/issues/41) are still not solved; solved: [#29](https://github.com/DronMDF/zold-lazy-node/issues/29).

**Comment by 0pdd** [comment]
@DronMDF 4 puzzles [#30](https://github.com/DronMDF/zold-lazy-node/issues/30), [#31](https://github.com/DronMDF/zold-lazy-node/issues/31), [#50](https://github.com/DronMDF/zold-lazy-node/issues/50), [#51](https://github.com/DronMDF/zold-lazy-node/issues/51) are still not solved; solved: [#29](https://github.com/DronMDF/zold-lazy-node/issues/29), [#41](https://github.com/DronMDF/zold-lazy-node/issues/41).

**Comment by 0pdd** [comment]
@DronMDF 3 puzzles [#31](https://github.com/DronMDF/zold-lazy-node/issues/31), [#50](https://github.com/DronMDF/zold-lazy-node/issues/50), [#51](https://github.com/DronMDF/zold-lazy-node/issues/51) are still not solved; solved: [#29](https://github.com/DronMDF/zold-lazy-node/issues/29), [#30](https://github.com/DronMDF/zold-lazy-node/issues/30), [#41](https://github.com/DronMDF/zold-lazy-node/issues/41).

**Comment by 0pdd** [comment]
@DronMDF 3 puzzles [#31](https://github.com/DronMDF/zold-lazy-node/issues/31), [#51](https://github.com/DronMDF/zold-lazy-node/issues/51), [#55](https://github.com/DronMDF/zold-lazy-node/issues/55) are still not solved; solved: [#29](https://github.com/DronMDF/zold-lazy-node/issues/29), [#30](https://github.com/DronMDF/zold-lazy-node/issues/30), [#41](https://github.com/DronMDF/zold-lazy-node/issues/41), [#50](https://github.com/DronMDF/zold-lazy-node/issues/50).

**Comment by 0pdd** [comment]
@DronMDF 7 puzzles [#31](https://github.com/DronMDF/zold-lazy-node/issues/31), [#55](https://github.com/DronMDF/zold-lazy-node/issues/55), [#59](https://github.com/DronMDF/zold-lazy-node/issues/59), [#60](https://github.com/DronMDF/zold-lazy-node/issues/60), [#61](https://github.com/DronMDF/zold-lazy-node/issues/61), [#62](https://github.com/DronMDF/zold-lazy-node/issues/62), [#63](https://github.com/DronMDF/zold-lazy-node/issues/63) are still not solved; solved: [#29](https://github.com/DronMDF/zold-lazy-node/issues/29), [#30](https://github.com/DronMDF/zold-lazy-node/issues/30), [#41](https://github.com/DronMDF/zold-lazy-node/issues/41), [#50](https://github.com/DronMDF/zold-lazy-node/issues/50), [#51](https://github.com/DronMDF/zold-lazy-node/issues/51).

**Comment by 0pdd** [comment]
@DronMDF 6 puzzles [#55](https://github.com/DronMDF/zold-lazy-node/issues/55), [#59](https://github.com/DronMDF/zold-lazy-node/issues/59), [#60](https://github.com/DronMDF/zold-lazy-node/issues/60), [#61](https://github.com/DronMDF/zold-lazy-node/issues/61), [#62](https://github.com/DronMDF/zold-lazy-node/issues/62), [#63](https://github.com/DronMDF/zold-lazy-node/issues/63) are still not solved; solved: [#29](https://github.com/DronMDF/zold-lazy-node/issues/29), [#30](https://github.com/DronMDF/zold-lazy-node/issues/30), [#31](https://github.com/DronMDF/zold-lazy-node/issues/31), [#41](https://github.com/DronMDF/zold-lazy-node/issues/41), [#50](https://github.com/DronMDF/zold-lazy-node/issues/50), [#51](https://github.com/DronMDF/zold-lazy-node/issues/51).

**Comment by 0pdd** [comment]
@DronMDF 5 puzzles [#55](https://github.com/DronMDF/zold-lazy-node/issues/55), [#59](https://github.com/DronMDF/zold-lazy-node/issues/59), [#61](https://github.com/DronMDF/zold-lazy-node/issues/61), [#62](https://github.com/DronMDF/zold-lazy-node/issues/62), [#63](https://github.com/DronMDF/zold-lazy-node/issues/63) are still not solved; solved: [#29](https://github.com/DronMDF/zold-lazy-node/issues/29), [#30](https://github.com/DronMDF/zold-lazy-node/issues/30), [#31](https://github.com/DronMDF/zold-lazy-node/issues/31), [#41](https://github.com/DronMDF/zold-lazy-node/issues/41), [#50](https://github.com/DronMDF/zold-lazy-node/issues/50), [#51](https://github.com/DronMDF/zold-lazy-node/issues/51), [#60](https://github.com/DronMDF/zold-lazy-node/issues/60).
